modules/user_accounts/php/NDB_Form_user_accounts.class.inc

```diff
@@ -550,5 +590,18 @@ class NDB_Form_user_accounts extends NDB_Form
 
         return null;
     }
+
+    function getDescriptionUsingPermID($permID)
+    {
+        $db =& Database::singleton();
```

Comment:
It's best to use NDB_Factory instead of singleton() to get an instance of Database class